projects/hipblaslt/tensilelite/Tensile/Tests/unit/test_CustomSchedule.py

```diff
@@ -573,6 +573,30 @@ def test_schedule_192x256x32_TF32(self):
         valid, message = isValid(schedule_info, {"kernel": kernel})
         assert valid, message
 
+    def test_schedule_128x256x32_TF32(self):
+        """Tests the 128x256x32 TF32 TN schedule."""
+        kernel = create_base_kernel()
+        kernel["ProblemType"].update({
+            "TransposeA": True, "TransposeB": False
+        })
+        kernel.update({
+            "UseF32XEmulation": True,
```

Comment:
```suggestion
            "UseF32XEmulation": True, "ForceUnrollSubIter": True,
```

The force unroll is required, existing example is also missing it, I am addressing that,